browse/controllers/stats_page/__init__.py

```diff
@@ -140,11 +140,18 @@ def get_monthly_submissions_page() -> Response:
 def get_submission_stats_csv() -> Response:
     """Get submission stats in CSV format."""
     csv_head = "month,submissions,historical_delta\n"
+    current_date = date.today()
     try:
+        rows = get_monthly_submission_stats()
         csv_data = "".join([
             f"{r.ym.strftime('%Y-%m')},{r.num_submissions},{r.historical_delta}\n"
-            for r in get_monthly_submission_stats()
+            for r in rows
         ])
+        last_row = rows[-1:]
```

Comment:
Kind of a minor nit, but why is this a slice?

Reply:
The current month's data may not be in the resultset, so the last row is used to determine whether current month needs to be fetched elsewhere.

Reply:
Right. Just seems a little funky to use a slice rather than just grabbing the last item (row) in the list; ``last_row`` ends up as a list with a single element.

Reply:
Oh gotcha -- yes, of course.